.github/workflows/support.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for support-requests - https://github.com/dessant/support-requests
+name: 'Support Requests'
+
+on:
+  issues:
+    types: [labeled, unlabeled, reopened]
+
+permissions:
+  issues: write
+
+jobs:
+  mark-support:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/support-requests@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Label used to mark issues as support requests
+          support-label: 'support'
+          # Comment to post on issues marked as support requests. Add a link
+          # to a support page, or set to `false` to disable
+          issue-comment: >
```

Comment:
```suggestion
          issue-comment: |
```

This should fix the new lines not being applied on the comment. I noticed it when creating the PR on the X repo.

Reply:
![image](https://user-images.githubusercontent.com/4512430/141104792-34b446c0-bf0b-497f-9184-f68b2aa38587.png)
vs 
![image](https://user-images.githubusercontent.com/4512430/141104891-769f2520-440c-464b-b380-cfe7e8dd8970.png)